<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>614</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>716</width>
    <height>614</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>716</width>
    <height>614</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Geek Chat</string>
  </property>
  <property name="windowOpacity">
   <double>2.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="contactList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>256</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="messagesList">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>421</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="msgButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>530</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="msgEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>530</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="chatInfoLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>2</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Chat</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="profileButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Profile</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to contacts</string>
    </property>
   </widget>
   <widget class="QWidget" name="profileWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>261</width>
      <height>521</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QLineEdit" name="loginEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="loginLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="passwordLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="statusLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
    <widget class="QComboBox" name="statusComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string>online</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>470</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>470</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QListView" name="filterUsersList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>256</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="delButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Del contact</string>
    </property>
   </widget>
   <widget class="QWidget" name="loginWidget" native="true">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>431</width>
      <height>521</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-9</number>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QPushButton" name="connectButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>420</y>
       <width>181</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CONNECT</string>
     </property>
    </widget>
    <widget class="QListView" name="serverMessagesList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>431</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>loginEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>statusbar</receiver>
   <slot>showMessage(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>603</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
